src/components/View.jsx

```diff
@@ -37,7 +39,8 @@ const View = (props) => {
           )}
         <blockquote
           cite={data.cite}
-          className={cx(!isEditMode && data.quotationLanguage)}
+          lang={data.quotationLanguage}
+          siteLang={language}
```

Comment:
@sneridagh what do you think of the naming? I think its a bit critical but I don't have a better idea to describe it, I am open for suggestions :)